plans/git_reference.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python
+
+import subprocess
+from pathlib import Path
+
+import fmf
+
+tree_root = Path.cwd().absolute()
+node = fmf.Tree(tree_root).find("/plans")
+with node as data:
+    data["discover"]["repository"] = "https://github.com/packit/specfile.git"
```

Comment:
Use `url` instead of `repository`. The latter is only supported for backwards-compatibility. I am not sure if or when it will be removed but the [tmt docs](https://tmt.readthedocs.io/en/stable/spec/plans.html#fmf) only mention `url` so I'd go with that to be safe for the future.

Reply:
My bad, I saw it in some example and assumed that it's `url` that is deprecated (or less specific).